python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
         'RefactoringChecker':
-             {'R1710': 'Since this function can return a non-None value, you should '
-                       'explicitly write "return None" whenever None is returned by '
-                       'this function. (Possibly including possibly at the end of the function body.)'
+            {'R1710': 'Since this function can return a non-None value, you should '
+                      'explicitly write "return None" whenever None is returned by '
+                      'this function. (Possibly including at the end of the function body.)'
              }
     },
     'pylint.checkers.design_analysis': {
-        'MisdesignChecker': {'R0913': 'Too many parameters (%s/%s)'}
+        'MisdesignChecker': {'R0912': 'The function has too many branches (%s out of a default limit '
+                                      'of %s). You should try to reduce the complexity of the function '
+                                      'by splitting it up.',
+                             'R0913': 'The function has too many parameters (%s out of a default limit '
+                                      'of %s). You should try to reduce the complexity of the function '
+                                      'by splitting up it, or combining related objects as a single one.',
+                             'R0914': 'The function has too many local variables (%s out of a default limit '
+                                      'of /%s).',
+                             'R0915': 'The function or method has too many statements (%s out of a default limit '
+                                      'of /%s). It should be split up into smaller functions or methods.',
+                             'R0916': 'The if statement contains too many boolean expressions '
+                                      '(%s out of a default limit of %s). You should see if you can reduce '
+                                      'the amount of expressions, or split it into branches.',
+                             }
+    },
+    'pylint.checkers.classes': {
+        'ClassChecker': {'E0203': 'You can\'t use class member %r prior to it\'s assignment '
+                                  'on line %s.',
+                         'E0211': 'Every class method requires at least one parameter, '
+                                  'which we conventionally name \'self\'.',
+                         'E0213': 'The first parameter of a method should always be called \'self\'. '
+                                  'This is a strongly adhered to convention.',
+                         'E0241': 'The class %r should not inherit from the same class multiple times.',
+                         'R0201': 'Method does not make use of \'self\', so you should rewrite the '
+                                  'method as a function and move it outside the class.',
+                         'R0205': 'Class %r inherits from \'object\', which can be omitted in Python3.',
```

Comment:
No need to mention Python 3 (PyTA only supports Python 3).

"All classes inherit from `object` by default, so you do not need to specify this in the class header."